python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```
@@ -1259,6 +1259,20 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.transceiver.media.override is arista.avd.defined %}
    transceiver media override {{ ethernet_interface.transceiver.media.override }}
 {%     endif %}
+{%     if ethernet_interface.transceiver.application_override_100gbase_srbd is arista.avd.defined(true) %}
+   transceiver application override 100gbase-srbd
+{%     else %}
+{%         if ethernet_interface.transceiver.application_override is arista.avd.defined %}
+   transceiver application override {{ ethernet_interface.transceiver.application_override }}
+{%         endif %}
+{%         for application_override in ethernet_interface.transceiver.application_override_lanes | arista.avd.default([]) %}
+{%             set application_override_cli = "transceiver application override " ~ application_override.override ~ " lanes start " ~ application_override.first_lane %}
+{%             if application_override.last_lane is arista.avd.defined %}
```

Should we add a check that `last_lane` is >= `first_lane`? CLI accepts contradicting values but only keeps start value.

```
veos-4-33-0f(config-s-s71-if-Et1)#transceiver application override 1 lanes start 2 end 1
veos-4-33-0f(config-s-s71-if-Et1)#show session-config diffs 
--- system:/running-config
+++ session:/s71-session-config
interface Ethernet1
+   transceiver application override 1 lanes start 2
veos-4-33-0f(config-s-s71-if-Et1)#
veos-4-33-0f(config-s-s71-if-Et1)#
veos-4-33-0f(config-s-s71-if-Et1)#transceiver application override 1 lanes start 2 end 2
veos-4-33-0f(config-s-s71-if-Et1)#show session-config diffs
--- system:/running-config
+++ session:/s71-session-config
interface Ethernet1
+   transceiver application override 1 lanes start 2 end 2
veos-4-33-0f(config-s-s71-if-Et1)#
```

---

we usually don't do this even if you are correct given that we can't do it in schema and the only (correct) option we have in template is to silently skip. So let it be for this one :)